cli/Cargo.toml

```diff
@@ -6,10 +6,13 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-parser = {version= "*", path = "../parser/"}
-core = {version= "*", path = "../core/"}
+parser = {version= "*", path = "../parser/" }
+core = {version= "*", path = "../core/", default-features = false, features = ["native"]}
```

Comment:
Since the CLI is a executable and not a library I think it would be more conventional not to create a separate feature flag for this crate but instead just use the defaults.

Reply:
I strongly think that it is reasonable to opt out of bundled packages in the CLI. The executable size grows by 50% at the moment, and when adding more packages it will grow even more, and some people may just prefer to use the package manager to download the packages they need rather than having more than they'll ever use, taking up space in the binary and giving slower startup times. So I believe there is a reason to have two distributions, one with bundled packages and one without, so I am keeping it as-is and hope you will agree with me :)